tests/integration/components/bs-modal-test.js

```diff
@@ -312,5 +312,31 @@ module('Integration | Component | bs-modal', function (hooks) {
       await settled();
       assert.dom('.modal').doesNotExist();
     });
+
+    test('it passes along fullscreen class attribute', async function (assert) {
```

Comment:
Small proposal to improve wording for clarity:

```suggestion
    test('it allows to render modal in fullscreen using @fullscreen argument', async function (assert) {
```

Reply:
Would be great if you could add a test that none of the classes is applied if `@fullscreen` argument is not passed.

Reply:
It seems that this test is currently grouped under the `it animates opening and closing the modal`. Would be great if you could move it above into the main module.

Reply:
1. Fixed
2. I do not understand what you are asking...

Reply:
The test suite uses `modules` to group related tests. The main module is declared very at the top of the file: https://github.com/kaliber5/ember-bootstrap/blob/e24988a19d7b21607ee7f386b49ce3f9b0159b0c/tests/integration/components/bs-modal-test.js#L11 But it contains an additional module declared later focused on the animation: https://github.com/kaliber5/ember-bootstrap/blob/e24988a19d7b21607ee7f386b49ce3f9b0159b0c/tests/integration/components/bs-modal-test.js#L251 You have put your new test within that module. It would be better if you could move your test out of that sub-module. So basically moving it above line 251.

Reply:
Exact. Understood
**Fixed**